activiti-steps/activiti-steps-assertions-api/src/main/java/org/activiti/steps/matchers/ThrowEventMatchers.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps.matchers;
+
+public class ThrowEventMatchers extends ActivityMatchers {
+
+    private static final String THROW_EVENT = "throwEvent";
+
+    private ThrowEventMatchers(String definitionKey) {
+        super(definitionKey);
+
+    }
+
+    @Override
+    public String getActivityType() {
+        return THROW_EVENT;
+    }
+
+    public static ThrowEventMatchers throwEvent(String definitionKey) {
```

Comment:
is this something like "hasBeenThrown()" ?

Reply:
This refers to a node in the process definition of type bpmn throw event. For instance:

```
 processOperations.start(ProcessPayloadBuilder
                                        .start()
                                        .withProcessDefinitionKey("broadcastSignalEventProcess")
                                        .build())
                .expect(
                        processInstance()
                                .hasBeenStarted(),
                        startEvent("startevent1")
                                .hasBeenCompleted(),
                        sequenceFlow("flow5")
                                .hasBeenTaken(),
                        throwEvent("signalintermediatethrowevent1")
                                .hasBeenCompleted(),
                        sequenceFlow("flow4")
                                .hasBeenTaken(),
                        endEvent("endevent1")
                                .hasBeenCompleted(),
                        processInstance()
                                .hasBeenCompleted()
                );
```

Reply:
yeah.. that make sense.. but semantically we should have something like:
```
throwEvent("signalintermediatethrowevent1").hasBeenThrown()
```
I do understand that the hasBeenCompleted() refers to the ACTIVITY_COMPLETED event.